### PR TITLE
Hotfix/document ui

### PR DESCRIPTION
**Pull request by basil7vals**

### CODE REVIEW

**Comment by basil7vals** [comment]
Cycle - 0 completed

**Comment by abdullah-siddiqui-git** [comment]
Cycle - 1 reviewed.

File: app/views/issues/_issues.html.erb [hunk 1]
```diff
@@ -5,9 +5,11 @@
     <% if request.path.include?('issues') %>
       <td><%= link_to issue.project.title, project_path(issue.project_id) %></td>
     <% end %>
-    <td><%= link_to issue.assignee.try(:name), member_path(issue.assignee) %></td>
-    <td><%= issue.issue_type.name %></td>
-    <td><%= issue.issue_state.name %></td>
+    <% if issue.assignee.present? %>
```
Comment (Anmol79) [inline app/views/issues/_issues.html.erb]: wouldn't this make the view inconsistent? as the column wouldn't be rendered for the issue that doesn't have any assignee.

File: config/locales/en.yml [hunk 2]
```diff
@@ -24,7 +24,7 @@ en:
     create:
       success: 'Team created Successfully'
     destroy:
-      success: 'Successfully deleted team!'
+      success: 'Successfully deleted team'
```
Comment (Anmol79) [inline config/locales/en.yml]: have consistency
most of the messages are like ```Team created successfully``` it should also be like ```Team deleted successfully```

File: public/system/documents/documents/000/000/010/original/staruml_install.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+cd ~/Downloads
```
Comment (Anmol79) [inline public/system/documents/documents/000/000/010/original/staruml_install.txt]: add this folder in git ignore. every uploaded document shouldn't be a part of the PR.

File: app/views/issues/_issues.html.erb (hunk 1 above)
Comment (basil7vals) [reply app/views/issues/_issues.html.erb]: I have re-checked again. It ain't causing problem. In fact, printing empty <td></td>

File: app/views/issues/_issues.html.erb [hunk 4]
```diff
@@ -2,12 +2,14 @@
 <% issues.each do |issue| %>
   <tr>
     <td><%= link_to issue.title, project_issue_path(issue.project_id, issue.id) %></td>
-    <% if request.path.include?('issues') %>
+    <% if request_path_issues %>
```
Comment (Anmol79) [inline app/views/issues/_issues.html.erb]: render_project_title

**Pull request merged**